packages/expo-location/src/Location.ts

```diff
@@ -161,6 +161,9 @@ let googleApiKey;
 const googleApiUrl = 'https://maps.googleapis.com/maps/api/geocode/json';
 
 export async function getProviderStatusAsync(): Promise<ProviderStatus> {
+  if (!ExpoLocation.getProviderStatusAsync) {
+    throw new UnavailabilityError('expo-location', 'getProviderStatusAsync');
```

Comment:
Are you sure we want `expo-location` to be here?
Let's see at the error message of `UnavailabilityError`: 
https://github.com/expo/expo/blob/d8dcfb650aa70f7ed9e25e2a406d5fc39fd09277/packages/%40unimodules/react-native-adapter/src/errors/UnavailabilityError.ts#L9-L16
So it expects module's name instead of package's name 😉